docs/content/docs/tutorials/emr-on-eks-example.md

```diff
@@ -0,0 +1,494 @@
+---
+title: "Run Spark jobs using EMR on EKS controller"
+description: "ACK service controller for EMR on EKS enables customers to run spark jobs on EKS clusters"
+lead: "Run Spark jobs using ACK service controller for EMR on EKS."
+draft: false
+menu:
+  docs:
+    parent: "tutorials"
+weight: 47
+toc: true
+---
+We are excited to announce ACK service controller for EMR on EKS. Customers have told us that they like the declarative way of managing resources on EKS clusters. Using ACK service controller for EMR on EKS, customers now have the ability to define and run EMR jobs directly from their Kubernetes clusters. EMR on EKS manages the lifecycle of these jobs and it is [3.5 times faster than open-source Spark](https://aws.amazon.com/blogs/big-data/amazon-emr-on-amazon-eks-provides-up-to-61-lower-costs-and-up-to-68-performance-improvement-for-spark-workloads/) because it uses highly optimized EMR runtime  
+
+To get started, you can download the EMR on EKS controller image from [Amazon ECR](https://gallery.ecr.aws/aws-controllers-k8s/emrcontainers-controller) and run Spark jobs in minutes. ACK service controller for EMR on EKS is available as a *developer preview* and is not recommended for production use. To learn more, visit the [EMR on EKS documentation](https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/emr-eks.html).
+
+## Installation steps
+
+#### Prereqs
+Install these tools before proceeding:
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html)
+- `kubectl` - [the Kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+- `eksctl` - [the CLI for AWS EKS](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html)
+- `yq` - [YAML processor](https://github.com/mikefarah/yq)
+- `helm` - [the package manager for Kubernetes](https://helm.sh/docs/intro/install/)
+
+Configure AWS CLI with sufficient permissions to install EKS cluster. Please see [documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) for further guidance
+
+#### Install EKS cluster
+
+Let's export environment variables that are needed for the EMR on EKS cluster setup. Please copy and paste commands into terminal for faster provisioning
+```
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+export EKS_CLUSTER_NAME="ack-emr-eks"
+export AWS_DEFAULT_REGION="us-west-2"
+export AWS_REGION="us-west-2"
+```
```

Comment:
Do we need 2 regions?

Reply:
I've always used 2 variables for defining region. I can try it with one and update with commit